tests/platform_tests/test_link_down.py

```diff
@@ -158,20 +164,33 @@ def test_link_down_on_sup_reboot(duthosts, localhost, enum_supervisor_dut_hostna
     # RP doesn't have any interfaces, check all LCs' interfaces
     link_status_on_all_LC(duthosts, localhost, fanouts_and_ports, up=False)
 
-    time.sleep(MAX_TIME_TO_REBOOT)
+    # Wait for ssh port to open up on the DUT
+    wait_for_startup(duthost, localhost, 0, MAX_TIME_TO_REBOOT)
 
     dut_uptime = duthost.get_up_time()
     logger.info('DUT {} up since {}'.format(hostname, dut_uptime))
     rebooted = float(dut_uptime_before.strftime("%s")) != float(dut_uptime.strftime("%s"))
     assert rebooted, "Device {} did not reboot".format(hostname)
 
+    # Verify that the links are all LCs are up
+    check_interfaces_and_services_all_LCs(duthosts, conn_graph_facts, xcvr_skip_list)
```

Comment:
no need for 
check_interfaces_and_services_all_LCs(duthosts, conn_graph_facts, xcvr_skip_list)
and
link_status_on_all_LC(duthosts, localhost, fanouts_and_ports)

they will be checked BEFORE each test

Reply:
For test completeness, and not relying on sanity or other tests, I would rather leave it in the test itself - doesn't hurt - other than execution time. But, in a whole pipeline regression run this would add atmost 10 mins to ~30+ hr run.

Reply:
my point is this:
```
test_link_status_on_sup_reboot():
   check_interfaces_and_services_all_LCs -- OK, already there
   do reboot and check
   check_interfaces_and_services_all_LCs  -- no need, because next test will check this again anyway
test_link_status_on_host_reboot():
  check_interfaces_and_services_all_LCs  -- OK, already there
  do reboot and check
  check_interfaces_and_services_all_LCs  -- OK, you added
```

Reply:
but I'm ok with adding this in every test